netlogo-gui/src/main/ide/JumpToDeclarationAction.scala

```diff
@@ -0,0 +1,18 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.ide
+
+import java.awt.event.{ActionEvent, MouseEvent}
+import java.awt.Point
+import javax.swing.{AbstractAction, Action}
+
+import org.nlogo.editor.EditorArea
+
+class JumpToDeclarationAction() extends AbstractAction {
```

Comment:
I really like these small actions. They let you keep the other classes free of Swing logic while still telling Swing how to present the action to the user.